src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/redundantthis/InputRedundantThisAnonymousObjects.java

```diff
@@ -0,0 +1,68 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.redundantthis;
+
+public class InputRedundantThisAnonymousObjects {
```

Comment:
please add comment that describes used config for check. For example
```
/*
* Config = default
*/
```
or
```
/*
* Config 
* prop1 = false
* prop2 = true
*/
```

Reply:
I have some doubts about doing this, since configuration for check is done by test method and an input file can be used by multiple test methods (hence audited for multiple configurations).

You might therefore ask why I've still added `// violatation` and `// no violation` in input files.

Well, the only reason I did that was I thought It might be more convenient for reviewers to be able to see where violations are in input file, so they do not have to also look at `RedudantThisCheckTest` class.

I was planning to remove all comments in input files after they've been approved by reviewers.

Reply:
But if you still want me to add comments for configuration, I will.
Just let me know.

Reply:
@wilcoln usually it is better to maintain inputs that have exactly one config and only one test case for each input. 
Sharing of inputs is not very convenient.
So I can suggest to make one input (merge existing into one) for one test case.

Reply:
Done.